services/app-api/prisma/schema.prisma

```diff
@@ -43,18 +43,33 @@ model State {
 }
 
 model User {
-  id                 String             @id
-  createdAt     DateTime               @default(now())
-  updatedAt     DateTime               @default(now()) @updatedAt
-  givenName          String
-  familyName         String
-  email              String
-  role               Role
-  divisionAssignment Division?
-  stateAssignments   State[]
-  stateCode          String?
-  questions          Question[]
-  responses          QuestionResponse[]
+  id                    String             @id
+  createdAt             DateTime           @default(now())
+  updatedAt             DateTime           @default(now()) @updatedAt
+  givenName             String
+  familyName            String
+  email                 String
+  role                  Role
+  divisionAssignment    Division?
+  stateAssignments      State[]
+  stateCode             String?
+  questions             Question[]
+  responses             QuestionResponse[]
+  audits                UserAudit[]
+  updatedByAuditEntries UserAudit[]        @relation("UpdatedBy")
+}
+
+model UserAudit {
+  id              String      @id
+  createdAt       DateTime    @default(now())
+  updatedAt       DateTime    @default(now()) @updatedAt
+  modifiedUserId  String      @map("userId")
+  user            User        @relation(fields: [modifiedUserId], references: [id])
+  updatedBy       User        @relation("UpdatedBy", fields: [updatedByUserId], references: [id])
+  updatedByUserId String
+  action          AuditAction
+  description     String
+  priorValue      Json?
```

Comment:
See this [comment](https://github.com/Enterprise-CMCS/managed-care-review/pull/1594#issuecomment-1496522763) about `priorValue`